<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>399</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>495</width>
    <height>399</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>399</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>车牌识别系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource.qrc">
    <normaloff>:/icons/icons/icon.png</normaloff>:/icons/icons/icon.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 246, 232);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icons/icons/cha.png</normaloff>:/icons/icons/cha.png</iconset>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>391</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QTabWidget::tab-bar {  		/*整个最上面的tab栏*/
    alignment: left;  
	top:10px;
	left:30px;

	
} 

QTabWidget::pane{
	border:none;

 
}

QTabBar::tab{
	color:#333333;
	background:transparent;
	
	font-family:&quot;微软雅黑&quot;;
	font-size:14px;
	
	padding-left:-9px;
	padding-right:-9px;
	width:75px;
	height:30px;
	margin-left:0px;
	

}

QTabBar::tab:selected, QTabBar::tab:hover
{
	color:#618BE5;
	background:transparent;
	
	font-family:&quot;微软雅黑&quot;;
	font-size:14px;
	border-bottom:1px solid #618BE5;
}





</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Introduce">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>介绍</string>
    </attribute>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>360</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border:none;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'宋体'; font-size:12pt; font-weight:600; color:#000000;&quot;&gt;数据集说明&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10pt; color:#000000;&quot;&gt;    数据集采用开源数据集CCPD&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'宋体'; font-size:10pt; color:#000000;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>21</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_2{
	max-height: 48px;
	border-top-right-radius: 20px; /*右上角圆角*/
	border-bottom-left-radius: 20px; /*左下角圆角*/
	background-color: rgb(173, 223, 255);
}
#pushButton_2:hover{
	background-color: rgb(255,255,255);
	color: rgb(0,0,0);
}
#pushButton_2:press{
	
	padding-top:5px;
	padding-left:5px;
}
	


</string>
     </property>
     <property name="text">
      <string>数据集</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_4{
	max-height: 48px;
	border-top-right-radius: 20px; /*右上角圆角*/
	border-bottom-left-radius: 20px; /*左下角圆角*/
	background-color: rgb(173, 223, 255);
}
#pushButton_4:hover{
	background-color: rgb(255,255,255);
	color: rgb(0,0,0);
}
#pushButton_4:press{
	
	padding-top:5px;
	padding-left:5px;
}
	


</string>
     </property>
     <property name="text">
      <string>训练</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resource.qrc">:/icons/icons/箭头_向右_o.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>19</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resource.qrc">:/icons/icons/箭头_向左_o.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>21</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButton_3{
	max-height: 48px;
	border-top-right-radius: 20px; /*右上角圆角*/
	border-bottom-left-radius: 20px; /*左下角圆角*/
	background-color: rgb(173, 223, 255);
}
#pushButton_3:hover{
	background-color: rgb(255,255,255);
	color: rgb(0,0,0);
}
#pushButton_3:press{
	
	padding-top:5px;
	padding-left:5px;
}
	


</string>
     </property>
     <property name="text">
      <string>算法</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>360</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border:none;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'宋体'; font-size:12pt; font-weight:600; color:#000000;&quot;&gt;算法分析&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10pt; color:#000000;&quot;&gt;    采用ctpn进行文字检测，crnn进行文字识别&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>360</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border:none;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;训练过程&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    测试机与训练集&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="TextRank">
    <attribute name="title">
     <string>识别</string>
    </attribute>
    <widget class="QPushButton" name="recognize">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>250</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#recognize{
	max-height: 48px;
	border-radius:8px;
	
	background-color: rgb(249, 246, 232);
}

#recognize:hover{
	background-color: rgb(255, 255, 255);
	color: rgb(0,0,0);
}
#recognize:press{
	padding-top:5px;
	padding-left:5px;
}
	
</string>
     </property>
     <property name="text">
      <string>识别</string>
     </property>
    </widget>
    <widget class="QPushButton" name="keyfile">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#keyfile{
	max-height: 48px;
	border-radius:8px;
	
	background-color: rgb(249, 246, 232);
}

#keyfile:hover{
	background-color: rgb(255, 255, 255);
	color: rgb(0,0,0);
}
#keyfile:press{
	padding-top:5px;
	padding-left:5px;
}
	


</string>
     </property>
     <property name="text">
      <string>选择图片</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graph">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>331</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>431</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icons/icons/4.png);
border-top-left-radius:20px;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resource.qrc">:/icons/icons/4.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>-1</number>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
image: url(:/icons/icons/望远镜.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icons/icons/宇宙星球-21.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
image: url(:/icons/icons/绿色 叶子 节能.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>tabWidget</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
